### ISSUE TITLE
@turf/union console.log published to npm

### ISSUE THREAD

**Issue by tkloht** [Turfjs/turf#1313]
I use the `@turf/union` package from npm and noticed that it logs to console on each invocation.

I think this is just a debug log published to npm by accident - see the js currently available on npm here: https://unpkg.com/@turf/union@6.0.0/index.js

```
function union(polygon1, polygon2, options) {
    if (options === void 0) { options = {}; }
    var coords1 = invariant_1.getGeom(polygon1).coordinates;
    var coords2 = invariant_1.getGeom(polygon2).coordinates;
    var unioned = martinez.union(coords1, coords2);
    console.log(unioned); // <------- here
    if (unioned.length === 0)
        return null;
    if (unioned.length === 1)
        return helpers_1.polygon(unioned[0], options.properties);
    else
        return helpers_1.multiPolygon(unioned, options.properties);
}
```

In the [typescript source](https://github.com/Turfjs/turf/blob/master/packages/turf-union/index.ts) there is no log.

**Comment by rowanwins** [comment]
Hey @tkloht

Thanks for the report. v6 of `@turf/union` isn't really ready for the primetime, it probably shouldn't have been published sorry. If you can revert back to v5 that would be great. v6 is mostly alpha stuff at the moment.

Cheers

**Comment by tkloht** [comment]
Thanks for looking at this, I am using 5.1.5 now.
In case someone is interested, I did not run into any other issues with v6 besides some outdated documentation (see #1314 ).

I have to say though, it would be really great if you could just publish a 6.0.1 to fix this to avoid more people running into this.

**Comment by DenisCarriere** [comment]
Published `6.0.1`, no more console.log

Thanks for reporting this @tkloht 👍
